Пулл для сдачи проекта

SharipovMS opened the pull request.

Commit:
Добавление строчки для создания пулл реквеста

irisqul commented on conftest.py:
```diff
@@ -8,3 +8,4 @@ def browser():
     driver.get('https://qa-scooter.praktikum-services.ru/')
```
Нужно исправить: урл записан в файле data нужно переиспользовать его оттуда

irisqul commented on tests/test_order.py:
```diff
@@ -25,4 +25,4 @@ def test_order_click_low_button(self, browser):
         order_page.fill_form_one()
         order_page.click_button(LocatorsOrderPage.button_next)
         order_page.fill_form_two()
-        assert 'Номер заказа' in order_page.get_element_text(LocatorsOrderPage.order_is_processed)
```
Нужно исправить: по заданию просят проверить сценарии заказа самоката на двух наборах данных

irisqul commented on pages/base_page.py:
```diff
@@ -47,3 +47,4 @@ def current_get_url(self):
         return self.browser.current_url
 
 
```
Отлично: используется POM

Commit:
исправлены локаторы с цветом самоката.
в итоге переработаны 2 теста, под разные наборы данных, рандомные персональные данные, 1 набор с черным самокатом, 2 набор с серым самокатом
